src/site/stages/build/drupal/graphql/nodeStepByStep.graphql.js

```diff
@@ -51,9 +51,20 @@ fragment nodeStepByStep on NodeStepByStep {
       }
     }
   }
-  fieldRelatedLinks {
+  fieldRelatedInformation {
     entity {
-      ... listOfLinkTeasers
+      ... on ParagraphLinkTeaser {
+        fieldLink {
+          url {
+            path
+            routed
+          }
+          uri
+          title
+          options
+        }
+        fieldLinkSummary
+      }
```

Comment:
Think it's worth abstracting this into its own fragment to share across nodes?

Reply:
I could see that being useful! 🙂

Reply:
Oh one already existed! Updated 😎